src/execution/debug.ts

```diff
@@ -92,6 +99,38 @@ export class GaugeDebugger {
         }
     }
 
+    private updateConfigFromLaunchJson(configobject: ConfigObj) {
+        let launchJsonPath = path.resolve(this.projectRoot, ".vscode", "launch.json");
```

Comment:
we can perhaps use vscode's api to fetch `launch` configurations?

ref: https://code.visualstudio.com/api/references/vscode-api#WorkspaceConfiguration

from the docs:

```
// launch.json configuration
const config = workspace.getConfiguration('launch', vscode.workspace.workspaceFolders[0].uri);

// retrieve values
const values = config.get('configurations');
```

Reply:
Thanks for this update. Will check and update the code respectively.

Reply:
Can you please also tell me why the test get failed in MAC OS.